src/Emogrifier/CssInliner.php

```diff
@@ -275,59 +174,6 @@ private function getHeadElement()
         return $this->domDocument->getElementsByTagName('head')->item(0);
     }
 
-    /**
-     * Creates a DOM document from the given HTML and stores it in $this->domDocument.
-     *
-     * The DOM document will always have a BODY element and a document type.
-     *
-     * @param string $html
-     *
-     * @return void
-     */
-    private function createUnifiedDomDocument($html)
-    {
-        $this->createRawDomDocument($html);
-        $this->ensureExistenceOfBodyElement();
-    }
-
-    /**
-     * Creates a DOMDocument instance from the given HTML and stores it in $this->domDocument.
-     *
-     * @param string $html
-     *
-     * @return void
-     */
-    private function createRawDomDocument($html)
-    {
-        $domDocument = new \DOMDocument();
-        $domDocument->encoding = 'UTF-8';
-        $domDocument->strictErrorChecking = false;
-        $domDocument->formatOutput = true;
-        $libXmlState = \libxml_use_internal_errors(true);
-        $domDocument->loadHTML($this->prepareHtmlForDomConversion($html));
-        \libxml_clear_errors();
-        \libxml_use_internal_errors($libXmlState);
-        $domDocument->normalizeDocument();
```

Comment:
`AbstractHtmlProcessor` does not call `normalizeDocument()`.  Should it?

Reply:
Probably. It looks like this is not covered with tests yet. I'll create a PR for that.

Reply:
I haven't been able to come up with a regression test for the missing `normalizeDocument`. As it's recommended to use it, I'll create a PR anyway.

Reply:
#662